platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -38,20 +38,41 @@ async function defaultRouteInit({
 
   unsubscriptions.push(instanceAddedUnsubscribe);
 
-  const { unsubscribe: seriesAddedUnsubscribe } = DicomMetadataStore.subscribe(
-    DicomMetadataStore.EVENTS.SERIES_ADDED,
-    ({ StudyInstanceUID, madeInClient }) => {
-      const studyMetadata = DicomMetadataStore.getStudy(StudyInstanceUID);
-      if (!madeInClient) {
-        HangingProtocolService.run(studyMetadata);
-      }
+  const { unsubscribe: seriesAddedUnsubscribe } = DisplaySetService.subscribe(
+    DisplaySetService.EVENTS.DISPLAY_SETS_CHANGED,
```

Comment:
The trigger for hanging protocol has to be when the display sets are computed, because a viewport has a display set in it, not a series.  That allows leaving the display set logic in the extensions, which otherwise would need to be moved into the hanging protocol itself, making each one very fragile.

Reply:
This makes sense. 
Bring back the `madeInClient` stuff since you don't want this to run when a new displaySet has been created by the user (e.g., export SR). The `madeInClient` make sure this is `.run` only happens when it is coming from the server